docs/guides/src/main/server/logging.adoc

```diff
@@ -7,90 +7,100 @@ title="Configuring logging"
 |%d{xxx}|Date|Renders a date with the given date format string.String syntax defined by `java.text.SimpleDateFormat`
-|%e|Exception|Renders the thrown exception, if any.
+|%e|Exception|Renders any thrown exception that exists.
 |%h|Hostname|Renders the simple host name.
 |%H|Qualified host name|Renders the fully qualified hostname, which may be the same as the simple host name, depending on the OS configuration.
 |%i|Process ID|Renders the current process PID.
-|%m|Full Message|Renders the log message plus exception (if any).
+|%m|Full Message|Renders the log message plus any exception that exists.
 |%n |Newline|Renders the platform-specific line separator string.
 |%N|Process name|Renders the name of the current process.
 |%p|Level|Renders the log level of the message.
 |%r|Relative time|Render the time in milliseconds since the start of the application log.
-|%s|Simple message|Renders only the log message, without exception trace.
+|%s|Simple message|Renders only the log message without exception trace.
 |%t|Thread name|Renders the thread name.
 |%t{id}|Thread ID|Render the thread ID.
-|%z{<zone name>}|Time zone|Set the time zone of log output to <zone name>.
+|%z{<zone name>}|Time zone|Sets the time zone of log output to <zone name>.
 |====
 
-=== Set the logging format
-To set the logging format for a logged line, build your desired format template using the table above and run the following command:
+=== Setting the logging format
+To set the logging format for a logged line, perform these steps:
+
+. Build your desired format template using the table above
+. Enter the following command:
++
 <@kc.start parameters="--log-format=\"\'<format>\'\""/>
-Be aware that you need to escape characters when invoking commands containing special shell characters such as `;` using the CLI, so you might want to set it in the configuration file instead.
+
+Note that you need to escape characters when invoking commands containing special shell characters such as `;` using the CLI. Therefore, consider setting it in the configuration file instead.
 
 .Example: Abbreviate the fully qualified category name
+
 <@kc.start parameters="\"\'%d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%c{3.}] (%t) %s%e%n\'\""/>
+
 The example above abbreviates the category name, which could get rather long in some cases, to three characters by setting `[%c{3.}]` in the template instead of the default `[%c]`.
 
-== Configuring raw quarkus logging properties
-At the time of writing, the logging features of the quarkus based Keycloak are basic, yet powerful. Nevertheless, expect more to come and feel free to join the https://github.com/keycloak/keycloak/discussions/8870[discussion] at GitHub.
+== Raw quarkus logging properties
+At publication time for this guide, the logging features of the Quarkus-based Keycloak are basic, yet powerful. Nevertheless, expect more to come and feel free to join the https://github.com/keycloak/keycloak/discussions/8870[discussion] at GitHub.
```

Comment:
```suggestion
At publication time for this guide, the logging features of the Quarkus-based Keycloak are basic, yet powerful. Nevertheless, expect more to come and feel free to join the https://github.com/keycloak/keycloak/discussions/categories/keycloak-x-quarkus-distribution[discussion] at GitHub.
```

Reply:
The mentioned discussion is closed now, for successors of parts to come. I think we go better with a more generic link here to not have to update it over and over ;)